frontend/app/routes/$lang+/_public+/status+/myself+/index.tsx

```diff
@@ -0,0 +1,228 @@
+
+    if (hCaptchaEnabled && captchaRef.current) {
+      try {
+        const response = captchaRef.current.getResponse();
+        formData.set('h-captcha-response', response);
+      } catch (error) {
+        /* intentionally ignore and proceed with submission */
+      } finally {
+        captchaRef.current.resetCaptcha();
+      }
+    }
+
+    fetcher.submit(formData, { method: 'POST' });
+  }
+
+  const errorSummaryId = 'error-summary';
+
+  const errorMessages = useMemo(() => {
+    const errors = fetcher.data && 'errors' in fetcher.data ? fetcher.data.errors : undefined;
+    return {
+      code: errors?.code?._errors[0],
+      sin: errors?.sin?._errors[0],
+    };
+  }, [fetcher.data]);
+
+  const errorSummaryItems = createErrorSummaryItems(errorMessages);
+
+  useEffect(() => {
+    if (hasErrors(errorMessages)) {
+      scrollAndFocusToErrorSummary(errorSummaryId);
+
+      if (hasErrors(errorMessages) && adobeAnalytics.isConfigured()) {
+        const fieldIds = createErrorSummaryItems(errorMessages).map(({ fieldId }) => fieldId);
+        adobeAnalytics.pushValidationErrorEvent(fieldIds);
+      }
+    }
+  }, [errorMessages]);
+
+  useEffect(() => {
+    if (fetcher.data && 'status' in fetcher.data) {
+      const targetElement = document.getElementById('status');
+      if (targetElement) {
+        targetElement.scrollIntoView({ behavior: 'smooth' });
+        targetElement.focus();
+      }
+    }
+  }, [fetcher.data]);
+
+  return (
+    <PublicLayout>
+      <div className="max-w-prose">
+        {fetcher.data && 'status' in fetcher.data ? (
+          <>
+            <ContextualAlert type={fetcher.data.status.alertType}>
+              <div>
+                <h2 className="mb-2 font-bold" tabIndex={-1} id="status">
+                  {t('status:myself.status-heading')}
+                </h2>
+                {fetcher.data.status.id ? getNameByLanguage(i18n.language, fetcher.data.status) : t('status:myself.empty-status')}
+              </div>
+            </ContextualAlert>
+            <ButtonLink id="cancel-button" variant="primary" type="button" routeId="$lang+/_public+/status+/index" params={params} className="mt-12">
+              {t('status:myself.check-another')}
+              <FontAwesomeIcon icon={isSubmitting ? faSpinner : faChevronRight} className={cn('ms-3 block size-4', isSubmitting && 'animate-spin')} />
+            </ButtonLink>
+          </>
+        ) : (
+          <>
+            {errorSummaryItems.length > 0 && <ErrorSummary id={errorSummaryId} errors={errorSummaryItems} />}
+            <fetcher.Form method="post" onSubmit={handleSubmit} noValidate autoComplete="off" data-gc-analytics-formname="ESDC-EDSC: Canadian Dental Care Plan Status Checker">
+              <input type="hidden" name="_csrf" value={csrfToken} />
+              {hCaptchaEnabled && <HCaptcha size="invisible" sitekey={siteKey} ref={captchaRef} />}
+              <p className="italic">{t('status:myself.form.complete-fields')}</p>
```

Comment:
Move the `Complete all fields.` paragraph outside the form and before the `ErrorSummary`.

https://www.w3.org/WAI/tutorials/forms/instructions/
> Provide such instructions before the `form` element to ensure that it is read aloud by screen readers before they switch to “Forms Mode”.